test/e2e/performance/README.md

```diff
@@ -0,0 +1,50 @@
+# Kubestellar performance regression testing
+This is a simple performance test that aims to identify major regressions. 
+
+The test adds a deployment to the WDS and measures the cumulative time it takes to create the WDS workload object, the Binding object, the ManifestWork object, and the deployment object on the WEC. The starting time is just prior to adding the workload to the WDS and the end time is just after the object is observed on the WEC. The timings are obtained from the objects creation timestamp which have seconds granuality. We expect this test to run on VMs and and anything below seconds is likely just noise. 
```

Comment:
This is good.
Just reading this, a careful reader will wonder: is it possible that the creation of the Binding was in response to the (presumably earlier, possibly only milliseconds earlier, although it is not explicitly mention in the current text here) creation of the BindingPolicy? Similar questions arise for the ManifestWork.

Reply:
That is an issue with the design of this test.

Reply:
This is moot now that the timer is started before the BindingPolicy is created.